src/content/advanced/spot-instances/_index.md

```diff
@@ -1,13 +1,13 @@
 ---
 title: Spot instances
-description: Spot instances (also called low-priority VMs on Azure) are a simple way to save on compute cost, if your use case permits it.
+description: Spot instances (also called low-priority VMs on Azure) are a simple way to save on compute cost, if your use case permits it. Here you find documentation for using them with Giant Swarm Kubernetes clusters, both on AWS and Microsoft Azure.
 weight: 40
 menu:
   main:
     identifier: advanced-spotinstances
     parent: advanced
 aliases:
-  - /basics/spot-instances/
+  - /advanced/spot-instances/
```

Comment:
- The alias you are deleting actually must stay there.
- The added alias does not make sense, as it is identical with the URI of the page.

Reply:
i think i fixed it